src/GenesisPreferencesWindow.cpp

```diff
@@ -35,6 +38,11 @@ GenesisPreferencesWindow::GenesisPreferencesWindow(BLooper* looper, BWindow *mai
 	m_SymlinkedPaths = new BCheckBox("followsymlinks", "Keep symlinked directory paths when navigating", new BMessage(PREFERENCES_CHANGED));
 	m_AskOnExit = new BCheckBox("askonexit", "Ask on exit", new BMessage(PREFERENCES_CHANGED));
 
+	m_EditorsMenu = new BMenu("choose ...",B_ITEMS_IN_COLUMN);
+	BMenuField *ChooseEditors = new BMenuField("",m_EditorsMenu);
+	ChooseEditors->SetDivider(0);
+
+	m_EditorApp = new BTextControl("editor", "F4 Editor application", "", NULL, B_WILL_DRAW|B_NAVIGABLE);
```

Comment:
```suggestion
	m_EditorApp = new BTextControl("editor", "Editor application (F4)", "", NULL, B_WILL_DRAW|B_NAVIGABLE);
```

That way the "F4" isn't that prominent; the user may have set the option not to show function keys.